pages/32-instructions.md

```diff
@@ -93,6 +93,11 @@ Different from traditional evaluations, note that the JSys Artifact Evaluation p
 
 Refer to the [Artifact Evaluation page](/artifact_evaluation) for details about the evaluation procedure, expectations, and tips for authors.
 
+### Referencing the Artefact
+
+The recommended method to reference an artifact is using a service like [Zenodo](https://about.zenodo.org/) to obtain a DOI and a persistent link.
```

Comment:
We miss something like "JSys recommends authors to publish their artifacts publicly and link to them in the final version of the manuscripts.